### PR TITLE
STF-5253 Add user reg attrs for pd & promo to api/v2

### PR DESCRIPTION
**Pull request by anatoly-cmdx**
https://instamart.atlassian.net/browse/STF-5251

### CODE REVIEW

File: source/includes/_users.md.erb [hunk 1]
```diff
@@ -93,7 +99,9 @@ curl '<%= config[:api_host] %>/users/test@test.com'
     "email": "test@test.com",
     "first_name": "Ivan",
     "last_name": "Petrov",
-    "location": "Moscow"
+    "location": "Moscow",
+    "privacy_terms": true
```
Comment (Grosefaid) [inline source/includes/_users.md.erb:103]: comma is missing

**Comment by Grosefaid** [comment]
/api/v2/sessions/#{access_token}/user is using Api::V2::UserSerializer - there will be changes too

File: source/includes/_users.md.erb [hunk 2]
```diff
@@ -10,7 +10,8 @@ curl '<%= config[:api_host] %>/users' \
   -d user[email]=#{EMAIL} \
   -d user[first_name]=#{FIRSTNAME} \
   -d user[last_name]=#{LASTNAME} \
-  -d user[password]=#{PASSWORD}
```
[the remark this answers is not in the document]
Comment (anatoly-cmdx) [reply source/includes/_users.md.erb:13]: fixed

File: source/includes/_users.md.erb (hunk 1 above)
Comment (anatoly-cmdx) [reply source/includes/_users.md.erb:103]: fixed

**Pull request merged**